From b69daf1bd7a46eac7910b72a1cb62806f68aa893 Mon Sep 17 00:00:00 2001
From: dgl <“dgl@rock-chips.com”>
Date: Mon, 28 Jul 2014 10:46:53 +0800
Subject: [PATCH] audio: rk3036 audio format check

change some scripts checkpatch ERROR and Warning,
do not care this version.

Signed-off-by: dgl@rock-chips.com
---
 arch/arm/boot/dts/rk3036-pinctrl.dtsi | 168 +++++++++++++-------------
 arch/arm/boot/dts/rk3036-sdk.dts      |  18 +--
 arch/arm/boot/dts/rk3036.dtsi         |  63 +++++-----
 sound/soc/rockchip/Kconfig            |  12 +-
 4 files changed, 131 insertions(+), 130 deletions(-)
 mode change 100755 => 100644 arch/arm/boot/dts/rk3036-pinctrl.dtsi

diff --git a/arch/arm/boot/dts/rk3036-pinctrl.dtsi b/arch/arm/boot/dts/rk3036-pinctrl.dtsi
old mode 100755
new mode 100644
index 5261973393d0..f369c3428c29
--- a/arch/arm/boot/dts/rk3036-pinctrl.dtsi
+++ b/arch/arm/boot/dts/rk3036-pinctrl.dtsi
@@ -3,7 +3,7 @@
 #include <dt-bindings/pinctrl/rockchip.h>
 #include <dt-bindings/pinctrl/rockchip-rk3036.h>
 
-/ { 
+/ {
 	pinctrl: pinctrl@20008000 {
 		compatible = "rockchip,rk3036-pinctrl";
 		reg = <0x20008000 0xA8>,
@@ -84,28 +84,28 @@
 				rockchip,pins = <UART0_SIN>,
 						<UART0_SOUT>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			uart0_cts: uart0-cts {
 				rockchip,pins = <UART0_CTSN>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			uart0_rts: uart0-rts {
 				rockchip,pins = <UART0_RTSN>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			uart0_rts_gpio: uart0-rts-gpio {
 				rockchip,pins = <FUNC_TO_GPIO(UART0_RTSN)>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 		};
 
@@ -114,8 +114,8 @@
 				rockchip,pins = <UART1_SIN>,
 						<UART1_SOUT>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 		};
@@ -125,8 +125,8 @@
 				rockchip,pins = <UART2_SIN>,
 						<UART2_SOUT>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 			/* no rts / cts for uart2 */
 		};
@@ -136,21 +136,21 @@
 			i2c0_sda:i2c0-sda {
 				rockchip,pins = <I2C0_SDA>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			i2c0_scl:i2c0-scl {
 				rockchip,pins = <I2C0_SCL>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			i2c0_gpio: i2c0-gpio {
 				rockchip,pins = <FUNC_TO_GPIO(I2C0_SDA)>, <FUNC_TO_GPIO(I2C0_SCL)>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 		};
 
@@ -158,21 +158,21 @@
 			i2c1_sda:i2c1-sda {
 				rockchip,pins = <I2C1_SDA>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			i2c1_scl:i2c1-scl {
 				rockchip,pins = <I2C1_SCL>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			i2c1_gpio: i2c1-gpio {
 				rockchip,pins = <FUNC_TO_GPIO(I2C1_SDA)>, <FUNC_TO_GPIO(I2C1_SCL)>;
                                 rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 		};
 
@@ -180,21 +180,21 @@
 			i2c2_sda:i2c2-sda {
 				rockchip,pins = <I2C2_SDA>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			i2c2_scl:i2c2-scl {
 				rockchip,pins = <I2C2_SCL>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			i2c2_gpio: i2c2-gpio {
 				rockchip,pins = <FUNC_TO_GPIO(I2C2_SDA)>, <FUNC_TO_GPIO(I2C2_SCL)>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 		};
 
@@ -202,36 +202,36 @@
 			spi0_txd:spi0-txd {
 				rockchip,pins = <SPI0_TXD>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			spi0_rxd:spi0-rxd {
 				rockchip,pins = <SPI0_RXD>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			spi0_clk:spi0-clk {
 				rockchip,pins = <SPI0_CLK>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			spi0_cs0:spi0-cs0 {
 				rockchip,pins = <SPI0_CS0>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 			spi0_cs1:spi0-cs1 {
 				rockchip,pins = <SPI0_CS1>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 
 		};
@@ -265,45 +265,45 @@
 			hdmi_gpio: hdmi-gpio {
 				rockchip,pins = <FUNC_TO_GPIO(HDMI_CEC)>, <FUNC_TO_GPIO(HDMI_SDA)>, <FUNC_TO_GPIO(HDMI_SCL)>, <FUNC_TO_GPIO(HDMI_HPD)>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				//rockchip,drive = <VALUE_DRV_DEFAULT>;				
+				//rockchip,drive = <VALUE_DRV_DEFAULT>;
 			};
-		};		
+		};
 
 		gpio1_i2s0 {
 			i2s0_mclk:i2s0-mclk {
 				rockchip,pins = <I2S0_MCLK>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 
 			i2s0_sclk:i2s0-sclk {
 				rockchip,pins = <I2S0_SCLK>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 
 			i2s0_lrckrx:i2s0-lrckrx {
 				rockchip,pins = <I2S0_LRCKRX>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 
 			i2s0_lrcktx:i2s0-lrcktx {
 				rockchip,pins = <I2S0_LRCKTX>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 
 			i2s0_sdo:i2s0-sdo {
 				rockchip,pins = <I2S0_SDO>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 
 			i2s0_sdi:i2s0-sdi {
 				rockchip,pins = <I2S0_SDI>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 
 			i2s0_gpio: i2s0-gpio {
@@ -314,7 +314,7 @@
 						<FUNC_TO_GPIO(I2S0_SDO)>,
 						<FUNC_TO_GPIO(I2S0_SDI)>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 		};
 
@@ -322,22 +322,22 @@
 			spdif_tx: spdif-tx {
 				rockchip,pins = <SPDIF_TX>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 			};
 		};
-		
+
 		gpio1_emmc0 {
 			emmc0_clk: emmc0-clk {
 				rockchip,pins = <EMMC_CLKOUT>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 
 			};
 
 			emmc0_cmd: emmc0-cmd {
 				rockchip,pins = <EMMC_CMD>;
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 
 			};
 
@@ -345,7 +345,7 @@
 			emmc0_bus1: emmc0-bus-width1 {
 				rockchip,pins = <EMMC_D0>;
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 
 			};
 
@@ -355,7 +355,7 @@
 					        <EMMC_D2 >,
 					        <EMMC_D3>;
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 
 			};
 		};
@@ -364,20 +364,20 @@
 			sdmmc0_clk: sdmmc0-clk {
 				rockchip,pins = <MMC0_CLKOUT>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 
 			};
 
 			sdmmc0_cmd: sdmmc0-cmd {
 				rockchip,pins = <MMC0_CMD>;
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 			};
 
 			sdmmc0_dectn: sdmmc0-dectn{
 				rockchip,pins = <MMC0_DETN>;
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 
 			};
 
@@ -385,7 +385,7 @@
 			sdmmc0_bus1: sdmmc0-bus-width1 {
 				rockchip,pins = <MMC0_D0>;
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 
 			};
 
@@ -395,7 +395,7 @@
 						<MMC0_D2>,
 						<MMC0_D3>;
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 
 			};
 
@@ -409,7 +409,7 @@
 					<GPIO1_C4>,  //D2
 					<GPIO1_C5>;  //D3
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 
 			};
 
@@ -456,7 +456,7 @@
 						<NAND_D5>,
 						<NAND_D6>,
 						<NAND_D7>;
-				rockchip,pull = <VALUE_PULL_DEFAULT>;	
+				rockchip,pull = <VALUE_PULL_DEFAULT>;
 
 			};
 
@@ -466,21 +466,21 @@
 			sdio0_clk: sdio0_clk {
 				rockchip,pins = <MMC1_CLKOUT>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 
 			};
 
 			sdio0_cmd: sdio0_cmd {
 				rockchip,pins = <MMC1_CMD>;
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 
 			};
 
 			sdio0_bus1: sdio0-bus-width1 {
 				rockchip,pins = <MMC1_D0>;
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 
 			};
 
@@ -490,7 +490,7 @@
 						<MMC1_D2>,
 						<MMC1_D3>;
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 
 			};
 
@@ -503,37 +503,37 @@
 				        <GPIO0_B5>,   //D2
 				        <GPIO0_B6>;   //D3
 				rockchip,pull = <VALUE_PULL_UP>;
-				
+
 
 			};
 		};
-                
+
 		gpio0_pwm{
 			pwm0_pin:pwm0 {
 				rockchip,pins = <PWM0>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 
 			};
 
 			pwm1_pin:pwm1 {
 				rockchip,pins = <PWM1>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 
 			};
 
 			pwm2_pin:pwm2 {
 				rockchip,pins = <PWM2>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 
 			};
 
 			pwm3_pin:pwm3 {
 				rockchip,pins = <PWM3(IR)>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
+
 
 			};
 		};
@@ -543,40 +543,40 @@
 				rockchip,pins = <MAC_CLKOUT>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
 			};
-		
+
 			mac_txpins: mac-txpins {
 				rockchip,pins = <MAC_TXD0>, <MAC_TXD1>,  <MAC_TXEN>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
-			
+
 			mac_rxpins: mac-rxpins {
 				rockchip,pins = <MAC_RXD0>, <MAC_RXD1>,<MAC_RXER>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
-			
+
 			mac_crs: mac-crs {
 				rockchip,pins = <MAC_CRS>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
-			
+
 			mac_mdpins: mac-mdpins {
 				rockchip,pins = <MAC_MDIO>, <MAC_MDC>;
 				rockchip,pull = <VALUE_PULL_DEFAULT>;
-				
-				
+
+
 			};
 		};
 
 
-		//to add 
+		//to add
+
 
-		
-	};	
+	};
 
 };
diff --git a/arch/arm/boot/dts/rk3036-sdk.dts b/arch/arm/boot/dts/rk3036-sdk.dts
index adfbc43b68a9..a6a79f6d8856 100644
--- a/arch/arm/boot/dts/rk3036-sdk.dts
+++ b/arch/arm/boot/dts/rk3036-sdk.dts
@@ -73,7 +73,7 @@
 
         status = "okay";
     };
-    
+
 	usb_control {
 		compatible = "rockchip,rk3036-usb-control";
 		host_drv_gpio = <&gpio2 GPIO_C7 GPIO_ACTIVE_LOW>;
@@ -86,11 +86,11 @@
 	key {
 		compatible = "rockchip,key";
 	};
-	
+
 	codec_hdmi_i2s: codec-hdmi-i2s {
 		compatible = "hdmi-i2s";
 	};
-	
+
 	rockchip-hdmi-i2s {
 		compatible = "rockchip-hdmi-i2s";
 		dais {
@@ -132,7 +132,7 @@
 
 &nandc {
 	status = "okay"; // used nand set "okay" ,used emmc set "disabled"
-};  
+};
 
 &nandc0reg {
 	status = "disabled"; // used nand set "disabled" ,used emmc set "okay"
@@ -151,11 +151,11 @@
 
         ignore-pm-notify;
 	keep-power-in-suspend;
-	
-	//poll-hw-reset 
+
+	//poll-hw-reset
 	status = "disabled";
 };
-    
+
 &sdmmc {
 		clock-frequency = <50000000>;
 		clock-freq-min-max = <400000 50000000>;
@@ -166,11 +166,11 @@
 
 		ignore-pm-notify;
         	keep-power-in-suspend;
-	
+
         	//vmmc-supply = <&rk808_ldo5_reg>;
 		status = "disabled";
 };
-		
+
 &sdio {
 		clock-frequency = <50000000>;
 		clock-freq-min-max = <200000 50000000>;
diff --git a/arch/arm/boot/dts/rk3036.dtsi b/arch/arm/boot/dts/rk3036.dtsi
index d9f420b24ac8..8266b96a50d8 100644
--- a/arch/arm/boot/dts/rk3036.dtsi
+++ b/arch/arm/boot/dts/rk3036.dtsi
@@ -398,6 +398,7 @@
 		clocks = <&clk_gates5 14>;
 		clock-names = "g_pclk_acodec";
 	};
+
 	spdif: spdif@10204000 {
 		compatible = "rockchip-spdif";
 		reg = <0x10204000 0x1000>;
@@ -422,38 +423,38 @@
 		status = "disabled";
 	};
 
-        pwm1: pwm@20050010 {
-                compatible = "rockchip,rk-pwm";
-                reg = <0x20050010 0x10>;
-                #pwm-cells = <2>;
-                pinctrl-names = "default";
-                pinctrl-0 = <&pwm1_pin>;
-                clocks = <&clk_gates7 10>;
-                clock-names = "pclk_pwm";
-                status = "disabled";
-        };
+	pwm1: pwm@20050010 {
+		compatible = "rockchip,rk-pwm";
+		reg = <0x20050010 0x10>;
+		#pwm-cells = <2>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&pwm1_pin>;
+		clocks = <&clk_gates7 10>;
+		clock-names = "pclk_pwm";
+		status = "disabled";
+	};
 
-        pwm2: pwm@20050020 {
-                compatible = "rockchip,rk-pwm";
-                reg = <0x20050020 0x10>;
-                #pwm-cells = <2>;
-                pinctrl-names = "default";
-                pinctrl-0 = <&pwm2_pin>;
-                clocks = <&clk_gates7 10>;
-                clock-names = "pclk_pwm";
-                status = "disabled";
-        };
+	pwm2: pwm@20050020 {
+		compatible = "rockchip,rk-pwm";
+		reg = <0x20050020 0x10>;
+		#pwm-cells = <2>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&pwm2_pin>;
+		clocks = <&clk_gates7 10>;
+		clock-names = "pclk_pwm";
+		status = "disabled";
+	};
 
-        pwm3: pwm@20050030 {
-                compatible = "rockchip,rk-pwm";
-                reg = <0x20050030 0x10>;
-                #pwm-cells = <2>;
-                pinctrl-names = "default";
-                pinctrl-0 = <&pwm3_pin>;
-                clocks = <&clk_gates7 10>;
-                clock-names = "pclk_pwm";
-                status = "disabled";
-        };
+	pwm3: pwm@20050030 {
+		compatible = "rockchip,rk-pwm";
+		reg = <0x20050030 0x10>;
+		#pwm-cells = <2>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&pwm3_pin>;
+		clocks = <&clk_gates7 10>;
+		clock-names = "pclk_pwm";
+		status = "disabled";
+	};
 
 	remotectl: pwm@20050030 {
 		compatible = "rockchip,remotectl-pwm";
@@ -536,7 +537,7 @@
 
 	    interrupts = <GIC_SPI 3 IRQ_TYPE_LEVEL_HIGH>,
                      <GIC_SPI 4 IRQ_TYPE_LEVEL_HIGH>,
-				     <GIC_SPI 5 IRQ_TYPE_LEVEL_HIGH>,
+			  	     <GIC_SPI 5 IRQ_TYPE_LEVEL_HIGH>,
                      <GIC_SPI 4 IRQ_TYPE_LEVEL_HIGH>;
 	    interrupt-names = "Mali_GP_IRQ",
 						  "Mali_GP_MMU_IRQ",
diff --git a/sound/soc/rockchip/Kconfig b/sound/soc/rockchip/Kconfig
index df792f1f0cb2..918a82eb8cec 100644
--- a/sound/soc/rockchip/Kconfig
+++ b/sound/soc/rockchip/Kconfig
@@ -117,7 +117,7 @@ config SND_RK_SOC_RT5631
 	help
 	  Say Y if you want to add support for SoC audio on rockchip
 	  with the RT5631.
-	  
+
 config SND_RK_SOC_RT5631_PHONE
 	tristate "SoC I2S Audio support for rockchip(phone) - RT5631"
 	depends on SND_RK_SOC
@@ -127,7 +127,7 @@ config SND_RK_SOC_RT5631_PHONE
 	  Say Y if you want to add support for SoC audio on rockchip
 	  with the RT5631.
 	  Driver code to use on the phone or voice Tablet.
-	  
+
 config SND_RK_SOC_RT5625
 	tristate "SoC I2S Audio support for rockchip - RT5625"
 	depends on SND_RK_SOC
@@ -160,7 +160,7 @@ config SND_RK_SOC_RT5640
 	select SND_SOC_RT5640
 	help
 	   RT5640 is pin to pin as RT5642, but not have dsp function.
-	  
+
 config SND_RK_SOC_RT3224
 	tristate "SoC I2S Audio support for rockchip - RT3224"
 	depends on SND_RK_SOC
@@ -168,7 +168,7 @@ config SND_RK_SOC_RT3224
 	select SND_SOC_RT3224
 	help
 	  RT3224 is pin to pin as RT3261, but not have dsp function.
-	  
+
 config SND_RK_SOC_RT5639
 	tristate "SoC I2S Audio support for rockchip - RT5639"
 	depends on SND_RK_SOC
@@ -176,7 +176,7 @@ config SND_RK_SOC_RT5639
 	select SND_SOC_RT5639
 	help
 	  rt5639
-	  
+
 config SND_RK_SOC_RT5616
 	tristate "SoC I2S Audio support for rockchip - RT5616"
 	depends on SND_RK_SOC
@@ -185,7 +185,7 @@ config SND_RK_SOC_RT5616
 	help
 	  Say Y if you want to add support for SoC audio on rockchip
 	  with the RT5616.
-	  
+
 config SND_RK_SOC_RT3261
 	tristate "SoC I2S Audio support for rockchip - RT3261"
 	depends on SND_RK_SOC
-- 
2.35.3

